src/server.c

```diff
@@ -584,7 +591,7 @@ dictType dbDictType = {
     NULL,                       /* key dup */
     NULL,                       /* val dup */
     dictSdsKeyCompare,          /* key compare */
-    dictSdsDestructorM,         /* key destructor */
+    dictSdsDestructorVar,         /* key destructor */
```

Comment:
cosmetics - but IMHO you could fix align in comment ( in simple word - remove two spaces)

Reply:
Done